test/unit/test-adapter.js

```diff
@@ -146,12 +118,104 @@ describe('SlackMessageAdapter', function () {
 
   describe('#expressMiddleware()', function () {
     beforeEach(function () {
-      this.adapter = new SlackMessageAdapter(workingVerificationToken);
+      this.adapter = new SlackMessageAdapter(workingSigningSecret);
+      this.next = sinon.stub();
+      this.dispatch = sinon.stub();
+      this.res = sinon.stub({
+        setHeader: function () { },
+        end: function () { }
+      });
+      this.errFn = sinon.stub();
     });
+
     it('should return a function', function () {
       var middleware = this.adapter.expressMiddleware();
       assert.isFunction(middleware);
     });
+    it('should error when body parser is used', function () {
```

Comment:
you need a `done` argument in here, and to call `done()` inside the `callsFake()` function to actually know this succeeded.